plugin.video.showcase.canada/default.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# KodiAddon 
+#
+from lib.scraper import myAddon
+import re
+import sys
+
+# Start of Module
+
+addonName = re.search('plugin\://plugin.video.(.+?)/',str(sys.argv[0])).group(1)
+ma = myAddon(addonName)
+ma.processAddonEvent()
+del myAddon
```

Comment:
you dont need to del your objects in python... it does garbage-collection automatically ;)